kura/org.eclipse.kura.wire.component.provider/src/main/java/org/eclipse/kura/internal/wire/common/DbServiceHelper.java

```diff
@@ -76,7 +76,6 @@ public static DbServiceHelper getInstance(final DbService dbService) {
      *             if argument is null
      */
     public void close(final Connection conn) {
-        requireNonNull(conn, message.connectionNonNull());
```

Comment:
Shouldn't it be a precondition to this method? Even though `DbService` `close()` method doesn't work on null elements, my proposition would be too add invariant to this method. I believe this is a good practice.